pixart.js

```diff
@@ -1 +1,30 @@
+var square = $('<div class="square"></div>');
 
+for (var i=0; i<8000; i++){
+  $('body').append(square.clone());
+  // console.log('so square broh');
```

Comment:
didn't know about this method. The docs on this are really interesting. I think in this particular case it doesn't matter because the element doesn't yet exist in the DOM. 👍